src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -34,15 +34,16 @@ export canopy_sw_rt_beer_lambert, # Radiative transfer
     compute_Kmm,
     optimal_co2_ratio_c3,
     intercellular_co2_pmodel,
-    gs_co2_pmodel,
-    gs_h2o_pmodel,
+    gcanopy_co2_pmodel,
```

Comment:
not sure what to call this. I am pretty sure this is canopy level, do we still want to call it stomatal conductance?